source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,619 @@
+created.
+
+The DHCP related rules will be provisioned after the respective group is
+created, and as part of the [create subnet](#creating-a-subnet-on-top-of-a-network)
+workflow.
+
+## Missing pieces
+
+### Ansible port module missing port_security_enabled attribute
+
+The [ansible module](https://docs.ansible.com/ansible/2.5/modules/os_port_module.html) currently does not support updating the *port_security_enabled* attribute for a port.
+
+To provide the user an integrated experience, ansible should allow for that attribute to be updated.
+
+A bug has been created to track this feature request. Its state can be followed
+[here](https://bugzilla.redhat.com/1623993).
+
+## Mapping networking API to OVN model objects
+
+The security group data will be modeled as a [port group](https://github.com/openvswitch/ovs/blob/master/ovn/ovn-nb.xml#L926).
+This ovn-nb table is only available on Open vSwitch 2.10, released upstream
+August 20th 2018.
+
+This new table maps a list of ACLs to a list of ports, thus eliminating
+the need to replicate the same ACL over and over, having the port in the
+match criteria as the only difference - this way, you match against a
+group of ports. More information can be found [here](https://docs.openstack.org/networking-ovn/latest/contributor/design/acl_optimizations.html) and [here](http://dani.foroselectronica.es/ovn-profiling-and-optimizing-ports-creation-434/).
+
+### Encoding the security group information
+
+The security group information will be encoded in the port group object.
+
+The security group has the following attributes:
+
+| name                 | type   | description |
+|----------------------|--------|-------------|
+| tenant_id       | string | The ID of the tenant.                                                      |
+| project_id      | string | The ID of the project.                                                     |
+| description (*) | string | A human-readable description for the resource. Default is an empty string. |
+| name            | string | Human-readable name of the resource.                                       |
+
+(*) - optional attributes
+
+All of the aforementioned attributes will be stored in the external_ids
+of the corresponding port group.
+
+### Encoding the security group rule information
+
+The security group information will be encoded in the ACL object, to
+which it naturally corresponds in the OVN-world.
+
+A security group rule has the following attributes:
+
+| name                 | type   | description |
+|----------------------|--------|-------------|
+| remote_group_id (**) | string | The remote group UUID to associate with this security group rule. You can specify either the  remote_group_id or remote_ip_prefix attribute in the request body. |
+| direction            | string | Ingress or egress, which is the direction in which the security group rule is applied.                                                                           |
```

Comment:
Why will be the `remote_group_id` not implemented, how is this solved in neutron?

Reply:
I'll make another PR to the feature page, updating it with information about the ```remote_group_id``` attribute.